pipeline_steps/github.groovy

```diff
@@ -14,15 +14,49 @@ def create_issue(
     sh """#!/bin/bash -xe
       cd ${env.WORKSPACE}
       . .venv/bin/activate
-      python rpc-gating/scripts/ghutils.py create_issue\
+      python rpc-gating/scripts/ghutils.py\
+        --org '$org'\
+        --repo '$repo'\
+        --pat '$pat'\
+        create_issue\
         --tag '$tag'\
         --link '$link'\
+        --label '$label'
+    """
+  }
+}
+
+/**
+ * Add issue link to pull request description
+ *
+ * Pull request commit messages include the issue key for an issue on Jira.
+ * Update the description of the current GitHub pull request with a link to
+ * the Jira issue.
+ */
+void add_issue_url_to_pr(){
```

Comment:
`def` says the function returns an object `void` says it returns nothing. `def` does not mean the same thing as it does in Python.